.github/workflows/NightlyBuildsCheck.yml

```
@@ -1,16 +1,34 @@
 name: Check Nightly Build Status
 on:
+  workflow_call:
+    inputs:
+      branch:
+        type: string
+      event:
+        type: string
+      should_publish:
+        type: string
   workflow_dispatch:
-  schedule:
-    - cron:  '0 8 * * *' # runs at 9am CET (summer) (replace the las * with 1-5 to trigger only on working days)
+    inputs:
+      branch:
+        type: string
+      event:
+        type: string
+      should_publish:
+        type: string
 
 permissions:
   contents: write
   issues: write
 
+concurrency:
+  group: status-${{ github.workflow }}-${{ github.ref_name }}
+  cancel-in-progress: true
```

```suggestion
  group: status-${{ github.workflow }}
  cancel-in-progress: false
```
Could you possibly check what's the syntax to say:
"queue the workflows?". I think `cancel-in-progress: false` + always having the same group should do it.

---

But why to queue them? That won't allow it to upload the latest report with the overwrite the old report named the same.